ckan_cloud_operator/providers/storage/s3/manager.py

```diff
@@ -0,0 +1,128 @@
+import json
+
+from ckan_cloud_operator import logs, kubectl
+from ckan_cloud_operator.config import manager as config_manager
+from ckan_cloud_operator.providers.cluster.manager import _generate_password
+from ckan_cloud_operator.providers.cluster.aws.manager import aws_check_output, get_aws_credentials, _config_get as aws_config_get
+
+from ..constants import CONFIG_NAME
+from .constants import PROVIDER_ID
+
+
+def initialize(interactive=False, storage_suffix='', use_existing_disk_name=False, dry_run=False):
+    default_zone = aws_config_get('aws-default-region', is_secret=True)
+    assert default_zone, 'No cluster region specified.'
+
+    if interactive and not dry_run:
```

Comment:
Why do we need separate credentials and region?

Reply:
and `storage-region` is not even used anywhere...

Reply:
@akariv separate credentials is used if you want to use separate AWS account with permissions limited only to S3.
`storage-region` is used to override default storage region (I updated the `create_bucket()` function)